app/models/illiad_openurl.rb

```diff
@@ -0,0 +1,35 @@
+###
+#  Class to handle creation of ILLiad OpenURL request
+###
+class IlliadOpenurl
+  def initialize(scan, redirect_url)
+    @scan = scan
+    @redirect_url = redirect_url
+  end
+
+  def illiad_page
+    { 'Action': '10', 'Form': '30', 'rft.genre': 'scananddeliver' }
+  end
+
+  def illiad_item_data
+    { 'rft.location': @scan.origin,
+      'rft.call_number': @scan.holdings.first.callnumber,
+      'rft.item': @scan.holdings.first.barcode }
```

Comment:
*sidenote:* We are (or were) DRY-ing up a bit of code in dealing with iterating through holdings when the item was persisted, but perhaps we should break out all holdings (for item selector purposes) and selected holdings (for success/status/email/mediation purposes) into separate methods.  I may have been trying to be too clever here.